presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java

```diff
@@ -1944,7 +1947,9 @@ private PhysicalOperation planGroupByAggregation(AggregationNode node, PhysicalO
                     hashChannel,
                     node.getGroupIdSymbol().map(mappings::get),
                     10_000,
-                    maxPartialAggregationMemorySize);
+                    maxPartialAggregationMemorySize,
+                    memoryLimitBeforeSpill,
+                    Optional.of(spillerFactory));
```

Comment:
`Optional.empty()` was used in tests that didn't use spilling. I have replaced that with some inlined implementation that `throw new UnsupportedOperatorException()`